src/shared/containers/Status/index.jsx

```diff
@@ -0,0 +1,95 @@
+import _ from 'lodash';
+import React from 'react';
+import PT from 'prop-types';
+import { connect } from 'react-redux';
+import actions from 'actions/status';
+import Tooltip from 'components/Tooltip';
+import APIErrorIcon from './APIError.svg';
+import NetworkErrorIcon from './NetworkError.svg';
+
+import './style.scss';
+
+function Tip({ errors, clear }) {
+  return (
+    <div styleName="Tip">
+      {
+        _.toPairs(errors).map(([url, message]) => (
+          <div key={url} styleName="item">
+            <div styleName="url">{url}</div>
+            <div styleName="message">{message}</div>
+            <a styleName="clear" onClick={() => clear(url)} role="button" tabIndex={-1}>&#10799;</a>
+          </div>
+        ))
+      }
+    </div>
+  );
+}
+
+Tip.propTypes = {
+  errors: PT.shape().isRequired,
+  clear: PT.func.isRequired,
+};
+
+function StatusContainer({
+  ApiErrors,
+  networkErrors,
+  clearApiErrorsStatus,
+  clearNetworkErrorsStatus }) {
+  if (_.isEmpty(ApiErrors) && _.isEmpty(networkErrors)) {
+    return null;
+  }
+  return (
+    <div styleName="container">
+      {!_.isEmpty(networkErrors) ?
```

Comment:
Also here, I would love to have it more generic, so that it is easy to add more error icon types in future, without having copy/paste the same blocks of code here.